packages/ruby/lib/readme/metrics.rb

```diff
@@ -58,7 +59,10 @@ def call(env)
     private
 
     def process_response(response:, env:, start_time:, end_time:)
-      har = Har::Serializer.new(env, response, start_time, end_time, @filter)
+      request = HttpRequest.new(env)
+      return if request.options?
```

Comment:
Thoughts on adding this to the if/else below? Maybe also logging that the request was ommited?

Reply:
In general, I try to avoid early returns